content/applications/general/email_communication/email_servers.rst

```diff
@@ -6,14 +6,29 @@ Odoo Online or Odoo.sh users
 ============================
 
 Since **Odoo sets up its own mail servers for the database**, outgoing and incoming emails already
-work out-of-the-box. So for **Odoo Online** and **Odoo.sh** customers, nothing needs to be done!
+work out-of-the-box. So for **Odoo Online** and **Odoo.sh** customers, nothing needs to be
+configured!
 
 Unless an external mail server is required to send large batches of mass mailing, simply use the
-standard online Odoo database normally since it has already been preconfigured for email.
+standard online Odoo database normally since it has already been preconfigured to send email.
+
+Default notifications system
+----------------------------
+
+Documents in Odoo (such as a CRM opportunity, a sales order, an invoice, etc.) have a discussion
+thread, called *chatter*.
+
+When a database user posts a message in the chatter, this message is sent by email to the followers
```

Comment:
sent by email to the followers of the record, except the sender (obviously)
-- dunno if worth to add tho

Reply:
I've added this in, thanks for pointing this out!